qiskit_pkg/MANIFEST.in

```diff
@@ -0,0 +1,2 @@
+include LICENSE.txt
+include README
```

Comment:
the readme file's a `.md`, right?

Reply:
This probably means we don't need the entry for the readme file because the package works explicitly including it. Looking at the terra manifest file the readme isn't listed there either.